.github/workflows/deploy-featurebranch.yml

```diff
@@ -2,12 +2,19 @@ name: deploy-featurebranch
 
 on:
   pull_request:
-    branches:
-      - master
+    types: [
+        labeled,
+        # default types below (https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request)
+        opened,
+        synchronize,
```

Comment:
Just to check - if we:
- tag a branch as feature branch
- push to that branch again

It will rebuild?

Reply:
Yes, it will. The action checks (on created, updated or labelled) if the PR has the `feature-branch` label. If so, it triggers the deploy